README.md

```diff
@@ -27,6 +27,24 @@ This behaviour can be disabled by this setting:
 spring.liquibase.enabled=false
 ```
 
+### Configure Keycloak
+Whereabouts uses Keycloak as an identity provider.
+The following settings need to be configured before running the application.
+
+| yaml key | description |
+|-----------|-------------|
+| `keycloak.auth-server-url` | keycloak url that used. e.g. `https://keycloak.example.com/auth` |
+| `keycloak.realm` | keycloak realm. |
+| `keycloak.resource` | keycloak client id. |
+
+
+There are 3 Keycloak client roles that need to be added for users to enable the access to all endpoints:
+- READ_INSTANCES,
```

Comment:
Is it possible to use the same style as Keycloak uses internally (e.g., `create-client`, `view-clients`), using lowercase and dash?

Reply:
And perhaps add a `whereabouts-` prefix to avoid conflicts with other tools?

Reply:
sure, I can change it. I followed the pattern we used for transmart client roles: "ROLE_ADMIN" etc., but maybe indeed lowercase-dash is better.

Reply:
Prefix not needed, because it is already scoped per client/resource, right?

Reply:
yes, these are roles (or rather authorities) per client, not per realm. I assumed that in the complete solution there will be a separate keycloak client per component (except transmart client that is already shared). Is that the plan? E.g. hyper-dicer will use a separate client within the same realm?

Reply:
The hyper-dicer needs the token to contact the transmart API, so there it may be different, unless we use impersonation there.